genesis/model/object/callable.go

```diff
@@ -19,22 +19,4 @@ package object
 // Callable allows itself to be called by its reference.
```

Comment:
This comment should be changed

Reply:
Any ideas?

Reply:
Callable is a marker interface that shows that object is able to be called